DiztinGUIsh/window/dialog/ExportDisassembly.Designer.cs

```diff
@@ -30,7 +30,7 @@ private void InitializeComponent()
         {
             System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(ExportDisassembly));
             this.cancel = new System.Windows.Forms.Button();
-            this.button2 = new System.Windows.Forms.Button();
```

Comment:
this is an unrelated change that crept in here. normally i'd remove it but, it won't hurt anything (just a rename)